assembl/static2/package.json

```diff
@@ -87,13 +87,15 @@
     "react": "^15.4.1",
     "react-apollo": "^1.4.12",
     "react-bootstrap": "^0.31.2",
+    "react-copy-to-clipboard": "^5.0.1",
```

Comment:
Before merging, we'll do another bundle size check to see how much these packages affected the size of our bundle. Hopefully not by much.